posthog/hogql/context.py

```diff
@@ -36,8 +36,8 @@ class HogQLContext:
     enable_select_queries: bool = False
     # Do we apply a limit of MAX_SELECT_RETURNED_ROWS=10000 to the topmost select query?
     limit_top_select: bool = True
-    # How many nested views do we support on this query?
-    max_view_depth: int = 1
+    # How many nested views do we support on this query? If `None`, no limit.
+    max_view_depth: int | None = 1
```

Comment:
Allow nested views.

Reply:
Did you mean to make this None/remove this and the check below in the resolver?